src/codegen/parser.cpp

```diff
@@ -384,6 +384,13 @@ AST_DictComp* read_dictcomp(BufferedReader* reader) {
     return rtn;
 }
 
+AST_Ellipsis* read_ellipsis(BufferedReader* reader) {
+    // TODO:
```

Comment:
Is there actually anything to do here?  I think we should avoid putting this in if it will silently handle ellipses the wrong way, but it looks like this might actually be all we need and we can remove the TODO?  Same for the serializer.